traja/models/train.py

```diff
@@ -245,15 +246,27 @@ def fit(
                     )
                 )
 
-            # Testing
-            if epoch % 10 == 9 and epoch != 0:
+            # Testing & Validation
+            evaluate_for_this_epoch = False
+            data_loader_to_evaluate = validation_loader
+            current_set = "Validation"
+            if epoch % validate_every == validate_every - 1 and epoch != 0:
```

Comment:
What happens if you should both test and validate? To me it looks like the code would only test.

Reply:
If a particular epoch happens to be a common multiple of `validate_every` and `test_every`, we only test for that epoch. Same is mentioned in the comments now.